src/coreclr/src/tools/aot/ILCompiler.Reflection.ReadyToRun/x86/CallPattern.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.	
```

Comment:
This change appears to be unintentional.

Reply:
Fixed, thank you.